PlotMarkersHandler.cls

```diff
@@ -0,0 +1,88 @@
+public with sharing class PlotMarkersHandler {
+    public static void plotMarkersMethodForAccountAndContact(Map<Id,sObject> mapForFetchingIds)
+    {
+           googleCallout(mapForFetchingIds.keyset());
+    }
+    @future(callout= True)
+    public static void googleCallout(Set<id> newRecordId1)
+    {
+        List<sObject> parametersList = new  List<sObject>();
+        List<Id> newRecordId = new list<Id>();
+        newRecordId.addAll(newRecordId1);
+        List<double> locationList = new List<double>();
+        String searchQuery;
+        List<String> listofLinks;
+        String googleMapEndPointUrl ='https://maps.googleapis.com/maps/api/geocode/json?sensor=false&address=';
+        if(String.valueOf(newRecordId[0]).substring(0,3) == '001')
```

Comment:
@Eternus-noopur Instead of this hardcode value you can use below code to get Sobject Type from id
`String.valueOf(newRecordId[0].getSObjectType()).equalsIgnoreCase(String.valueOf(Account.sObjectType))` and same for contact as well.